packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -36,6 +36,28 @@ const Color _kLightThemeSplashColor = const Color(0x66C8C8C8);
 const Color _kDarkThemeHighlightColor = const Color(0x40CCCCCC);
 const Color _kDarkThemeSplashColor = const Color(0x40CCCCCC);
 
+/// Configures the tap target and layout size of certain Material widgets.
+///
+/// The following widgets are impacted:
+///
+///   * [FloatingActionButton], only the mini tap target size is increased.
+///   * [MaterialButton]
+///   * [OutlineButton]
+///   * [FlatButton]
+///   * [TimePicker]
+///   * [SnackBar]
+enum MaterialTapTargetSize {
+  /// Expands the minimum tap target size to 48dp by 48dp.
```

Comment:
"dp" is not a type of unit we define in the context of Flutter's documentation.
Maybe just say "48 by 48 pixels" or something.

Reply:
Fixed